### PR TITLE
chore: enable new tree shaking for all tests.

### PR DESCRIPTION
**Pull request by IWANABETHATGUY**
## Require Documentation?

- [ ] No
- [ ] Yes, the corresponding rspack-website PR is \_\_

### CODE REVIEW

File: webpack-test/cases/parsing/issue-7519/test.config.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
+/**@type {import("@rspack/cli").Configuration} */
```
Comment (IWANABETHATGUY) [inline webpack-test/cases/parsing/issue-7519/test.config.js:1]: traced https://github.com/web-infra-dev/rspack/issues/4834

**Comment by ahabhgk** [comment]
Could you also enable this in `ConfigTestCases.template.js`

**Comment by IWANABETHATGUY** [comment]
> Could you also enable this in `ConfigTestCases.template.js`

Done

**Comment by IWANABETHATGUY** [comment]
!canary

**Review by ahabhgk — APPROVED** [review]

**Comment by IWANABETHATGUY** [comment]
!canary

**Pull request merged**